### PR TITLE
Full-text search translation

### PR DESCRIPTION
**Pull request by DebbieSan**
This PR translates the FullText  Search Box from English to French.

**These are the only changes I made:**

```
#: home/welcome.html:68
-#, fuzzy
 msgid "Try Fulltext Search"
-msgstr "Recherche en plein texte"
+msgstr "Essayez la recherche en plein texte"

 #: home/welcome.html:69
 msgid "Find matching results within the text of millions of books"
-msgstr ""
+msgstr "Trouvez des résultats correspondants dans le texte de millions de livres"
```

Any other changes are unrelated to this PR and from running the update/validate commands.

### CODE REVIEW

**Comment by cdrini** [comment]
Could you reopen this one? They will be much easier now that your other PR was merged! Sorry for the run around!

**Comment by DebbieSan** [comment]
@cdrini No worries. This has been completed. Thank you :)

File: openlibrary/i18n/fr/messages.po [hunk 1]
```diff
@@ -4416,7 +4415,7 @@ msgstr "Personne"
 
 #: search/advancedsearch.html:50
 msgid "Full Text Search"
-msgstr "Recherche en plein texte"
+msgstr "Essayez la recherche en plein texte"
```
Comment (cdrini) [inline openlibrary/i18n/fr/messages.po:4418]: This change looks unintentional

```suggestion
msgstr "Recherche en plein texte"
```

**Review by cdrini — APPROVED** [review]
Lgtm! Ty @DebbieSan

**Pull request merged**